projects/this-is-not-a-leaf/index.md

```diff
@@ -0,0 +1,125 @@
+
+![](./assets/algo1.png)
+![](./assets/algo2.png)
+![](./assets/algo3.png) 
+![](./assets/algo4.png)
+
+---------
+
+
+This first examplary task has been fullfilled correctly in almost all of the four cases. It has many constants and few, clear variables. But what made one person write down words outside of the square instead of drawing associations? It’s called inattention. A capricious factor when creating an algorithm. Because no matter how precise you are with your phrasing, if your subjects are not concentrating, the task may have way different results than expected. But now that the basics have been explained, let’s hop to the second attempt…
+
+----------
+
+### The Second Algorithm
+
+*Draw leaf-like forms (ovals that thin to the tails).*
+
+1. *Crisscross, in different sizes, as close as possible whilst still not touching!*
+1. *Fill some of them with hatchures, others with tiny dots and leave others empty.*
+*You do not have to fill the entire sheet!*
+
+In this case, the only constant is the shape oft he objects. Variables are the sheet format, the colour oft he chosen pencil, the number of drawed shapes, their exact location on the sheet, their exact size and, conditionally, the filling of each shape. As you will learn, to get your algorithm to be unmistakable it should be the other way around: the constants should predominate, the variables should be fewer and more precise:
+
+![](./assets/ergebnis2.png)
+![](./assets/ergebnis6.png)
+![](./assets/ergebnis7.png)
+
+The results show that this instruction often persuades subjects to draw entire leafs instead of the described leaf-like shapes.
+Therefore, another approach is to change the first sentence like this:
+
+*Draw ovals that thin to the tails.*
+
+This eliminates the thought of leafs from the subjects‘ mind and lets him focus on the described form.
+
+![](./assets/ergebnis5.png)
+![](./assets/ergebnis8.png)
+
+But still, people tend to draw the shapes comparative widely spread from each other. Hence, the first point in the instruction has to be changed - for example like this:
+
+*crisscross, in different sizes, almost touching.*
+
+Now, the subjects do not focus on the shapes not touching but rather on them being drawn closely.
+
+![](./assets/ergebnis3.png)
+![](./assets/ergebnis1.png)
+
+
+Also, the filling of the shapes has often been neither dotted, nor hatched, nor let empty. But in this case it’s important to focus on what you want as outcome. Do you really want the results to be all the same?
+Nah.
+So the second point has to be changed:
+
+*Fill some of them with lines or dots and leave others empty.*
+
+This way you get a wider range of possible correct outcomes while increasing the space for creativity.
+
+----------
+
+### The Revised Second Algorithm
+
+
+The revised algorithm now looks like this:
+
+*Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.*
```

Comment:
maybe better as quote:   

> Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.

done like this  

    > Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.